REVIEW_GUIDELINE.md

```diff
@@ -0,0 +1,255 @@
+# Guideline for Pull Request Review
+
+## Introduction
+
+Code review is an essential part of the software development life cycle and is also one of the most challenging and time-consuming aspects of the software development process, requiring experienced team members to spend time reading, thinking about, assessing, and responding to new feature, fix, and system implementations.
+
+Here are some guidelines to assist you in reviewing pull requests.
+
+## Guidelines
+
+Before jumping straight to the code changes, some obvious things should be addressed first.
+
+**Pull Request Checklist:**
+
+1. Commit Message (spell checks, commit lint etc.)
+2. Appropriate title of pull request
+3. Number of commits (should be one commit, unless required)
+4. Check if there should be single pull request or multiple pull requests
+5. Source branch and target branch of pull request
+6. Check if pipelines succeeded
```

Comment:
Can we please rephrase this sentence to something like `Check if recent changes did not break any existing functionalities and all the checks of the pipeline are passed` or anything you find more insightful